Presentation.Web/app/components/it-system/usage/tabs/it-system-usage-tab-data-processing.controller.ts

```diff
@@ -0,0 +1,17 @@
+﻿((ng, app) => {
+    app.config(["$stateProvider", $stateProvider => {
+        $stateProvider.state("it-system.usage.dataprocessing", {
+            url: "/dataprocessing",
+            templateUrl: "app/components/it-system/usage/tabs/it-system-usage-tab-data-processing.view.html",
+            controller: "systemUsage.DataProcessing"
+        });
+    }]);
+
+    app.controller("systemUsage.DataProcessing", ["$scope",
+        ($scope) => {
+            const itSystemUsage= $scope.usage;
+            $scope.dataProcessingRegistrations = itSystemUsage.associatedDataProcessingRegistrations;
+            $scope.filterDataProcessor = itSystemUsage.associatedDataProcessingRegistrations.length > 0;
```

Comment:
why use the "filter" word and why not just bind to the length of `$scope.dataProcessingRegistrations` ?
Adding additional properties just adds more maintenance